components/IscnRegisterForm.vue

```diff
@@ -582,17 +582,22 @@ export default class IscnRegisterForm extends Vue {
   }
 
   get exif() {
-    const extension = this.fileType!.split('/')
-    return {
-      ...this.exifInfo,
```

Comment:
Do we still want to have information from `this.fileType` and  `this.fileSize` even if no exif?
If yes just guard the `...this.exifInfo` with empty object